src/components/scenes/PinLoginScene.tsx

```diff
@@ -166,6 +167,7 @@ class PinLoginSceneComponent extends React.Component<Props, State> {
             onPress={this.handlePress}
           />
         )}
+        <SafeAreaView edges={['bottom']} />
```

Comment:
This will crash unless the user provides a `SafeAreaContext`. We either need to include that inside the `LoginUiProvider` component for completeness, or we need to be *very* clear in the Readme file that this is needed.

Reply:
done. worked great inside of `LoginUiProvider`